practice-one/src/components/SearchBox/index.tsx

```diff
@@ -1,12 +1,15 @@
 // react
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useCallback } from 'react';
 
 // components
 import Input from '@components/common/Input';
 
 // types
 import { StudentProfile } from 'src/types';
 
+import debounce from 'lodash/debounce';
```

Comment:
no need to use debounce from lodash
just need to import lodash to use debounce is unnecessary